ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/suites/CommandDeduplicationIT.scala

```diff
@@ -509,7 +522,20 @@ final class CommandDeduplicationIT(
             ),
             party,
           )
-        } yield {}
+        } yield {
+          assertDeduplicationOffset(
+            Ref.HexString.assertFromString(response.offset.getAbsolute),
+            response,
```

Comment:
Note to self: This test assumes inclusive deduplication offsets. Will need to change when we switch to exclusive dedup offsets.